lib/BackgroundJob.php

```diff
@@ -123,12 +137,45 @@ protected function createPublicity(Announcement $announcement, array $publicity)
 			->setObject('announcement', (string)$announcement->getId())
 			->setSubject('announced', [$announcement->getUser()]);
 
+		$template = $this->mailer->createEMailTemplate('announcementcenter::sendMail');
+		$template->setSubject($announcement->getSubject());
+		$template->addHeader();
+		$template->addHeading($announcement->getSubject());
+		$this->setMailBody($template, $announcement->getPlainMessage());
+		$template->addFooter();
+		$email = $this->mailer->createMessage();
+		$email->useTemplate($template);
+
 		$groups = $this->manager->getGroups($announcement);
 		if (\in_array('everyone', $groups, true)) {
-			$this->createPublicityEveryone($announcement->getUser(), $event, $notification, $publicity);
+			$this->createPublicityEveryone($announcement->getUser(), $event, $notification, $email, $publicity);
 		} else {
 			$this->notifiedUsers = [];
-			$this->createPublicityGroups($announcement->getUser(), $event, $notification, $groups, $publicity);
+			$this->createPublicityGroups($announcement->getUser(), $event, $notification, $email, $groups, $publicity);
+		}
+	}
+
+	/**
+	 * Special-treat list items and strip empty lines
+	 *
+	 * @param IEMailTemplate $template
+	 * @param string         $message
+	 *
+	 * @return void
+	 */
+	protected function setMailBody(IEMailTemplate $template, string $message): void {
+		$lines = explode("\n", $message);
+
+		foreach ($lines as $line) {
+			if (trim($line) === '') {
+				continue;
+			}
+
+			if (strpos(trim($line), '* ') === 0) {
+				$template->addBodyListItem(trim(substr($line, strpos($line, '*') + 1)));
```

Comment:
```suggestion
				$template->addBodyListItem(trim(substr($line, strpos($line, '*') + 1)), '', '', '', false);
```

This should solve the empty `()` in plain text mails

Reply:
Nice, changed now :ok_hand: